### PR TITLE
1.0.1a

### PR DESCRIPTION
**Pull request by naili-m**
- Added API based authentication to the existing basic auth

### CODE REVIEW

File: trend-micro-sms/operations.py [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        response = make_rest_call(endpoint, config, method='POST')
+        return response
+    except Exception as err:
+        logger.exception(str(err))
+        raise ConnectorError(str(err))
+
+
+def check_health(config):
+    try:
+        logger.error(config)
```
Comment (cs-pooja-singh) [inline trend-micro-sms/operations.py]: to be removed

Comment (naili-m) [reply trend-micro-sms/operations.py]: done

File: trend-micro-sms/info.json [hunk 2]
```diff
@@ -0,0 +1,313 @@
...
+                "description": "IP address or FQDN of the TrendMicro SMS server to which you will connect and perform the automated operations."
+            },
+            {
+                "title": "Authentication Type",
+                "required": true,
+                "editable": true,
+                "visible": true,
+                "type": "select",
+                "name": "authentication_type",
+                "value": "Basic Authentication",
```
Comment (cs-prerna-joshi) [inline trend-micro-sms/info.json]:
```suggestion
                "value": "Basic Auth",
```

**Review by cs-prerna-joshi — COMMENTED** [review]
Can you add release_notes.md file.

File: trend-micro-sms/info.json (hunk 2 above)
Comment (naili-m) [reply trend-micro-sms/info.json]: done

**Review by cs-prerna-joshi — APPROVED** [review]
LGTM

**Pull request merged**